pegasus/sites.v3/code.org/public/research.md

```diff
@@ -41,13 +41,6 @@ separately.
 * Findings show that after completing one Hour of Code activity students report liking computer science more, feel that they are better able to learn computer science, and are better at computer science than their peers.
 
 
-Other reports
-
-- [Code.org Evaluation Report 2015-16 (PDF)](/files/EvaluationReport2015-16.pdf)
```

Comment:
Just to confirm we want to remove this? Should `/files/EvaluationReport2015-16.pdf` also be removed?

Reply:
Luckily, it sounds like Dani is hoping to delete the section but keep the PDF (linked from an annual report) https://codedotorg.slack.com/archives/C0453TUMLE7/p1724190729875179. Thanks for checking!

Reply:
Sounds good!